external/nmea/info.h

```diff
@@ -116,6 +116,18 @@ typedef struct _nmeaINFO
   double  direction;  //!< Track angle in degrees True
   double  declination; //!< Magnetic variation degrees (Easterly var. subtracts from true course)
 
+  double  rms_pr;     //!< RMS value of the pseudorange residuals;
+
+  /**
+   * < Includes carrier phase residuals during periods of RTK (float) and RTK (fixed) processing
```

Comment:
Is this docstring for `satinfo`? If yes, the `<` prefix should be removed, the blank line 130 removed and the docstring on 131 removed.

Reply:
I think astyle messed up something, I'll check it